### PR TITLE
feat: admin notifications

### PR DESCRIPTION
**Pull request by izabellalarsson**
Showing notification when adding a vehicle or a booking.

### CODE REVIEW

**Review by lundeeell — CHANGES_REQUESTED** [review]
Looks good, just some smaller fixes! 👍

File: packages/engine-ui/src/components/Notifications.tsx [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+  React.useEffect(() => {
+    if (notifications.length > 0) {
+      setTimeout(() => {
+        handleOnClose(notifications[0].id)
+      }, 20000)
+    }
+  }, [notifications, updateNotifications])
+
+  const notificationType = (notification: NotificationType) => {
+    switch (true) {
```
[the remark this answers is not in the document]
Comment (izabellalarsson) [reply packages/engine-ui/src/components/Notifications.tsx:86]: Refactor so we switch on `notification.id.substr(0, 3)` thinking we will have more cases in the future so keeping the switch but I see what you mean!

File: packages/engine-ui/src/components/Notifications.tsx [hunk 2]
```diff
@@ -107,19 +112,13 @@ const Notifications: React.FC<{
...
-    updateNotifications((notifications: NotificationType[]) =>
-      notifications.filter((notification) => notification.id !== itemId)
-    )
-  }
-
   return (
-    <NotificationsContaioner>
-      {notifications.length > 0 &&
+    <NotificationsContainer>
+      {notifications &&
```
Comment (lundeeell) [inline packages/engine-ui/src/components/Notifications.tsx:117]: Unnecessary check

**Review by lundeeell — APPROVED** [review]
Looks good, however, I think nullable check could be removed

**Pull request merged**